### PR TITLE
Async tx submit

### PR DESCRIPTION
**Pull request by szerintedmi**

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 42592adc6d9e122691d2cc315c152dc1c6145480

https://app.netlify.com/sites/augmint/deploys/5ab3c0e2a6188f143bce5aac

File: src/containers/app/EthereumTxStatus.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                            />
+                        )}
+                    </MyListGroup.Row>
+                );
+            });
+
+        return !txList ? null : (
+            <Psegment>
+                <Container>{txList}</Container>
+                {/* <Modal open={true}>
```
Comment (reyrodrigez) [inline src/containers/app/EthereumTxStatus.js]: any reason not to remove the commented part?

File: src/containers/loan/collectLoan/index.js [hunk 2]
```diff
@@ -31,7 +31,6 @@ class CollectLoanMain extends React.Component {
                 this.props.augmintToken.isConnected)
         ) {
             // loanManager mounted
-            console.debug("fetch2");
```
Comment (reyrodrigez) [inline src/containers/loan/collectLoan/index.js]: `Fetch II - the fetch strikes back`

File: src/containers/loan/newLoan/NewLoanSuccess.js [hunk 3]
```diff
@@ -1,24 +1,11 @@
 import React from "react";
-import { Link } from "react-router-dom";
 import { EthSubmissionSuccessPanel } from "components/MsgPanels";
 
 export default function NewLoanSuccess(props) {
-    const { loanAmount, repaymentAmount, collateralEth, loanId, borrower, productId } = props.result;
+    const { collateralEth } = props.result;
     return (
-        <EthSubmissionSuccessPanel header={<h3>You've got a loan</h3>} result={props.result} dismissable={false}>
```
Comment (reyrodrigez) [inline src/containers/loan/newLoan/NewLoanSuccess.js]: `\`

File: src/containers/app/EthereumTxStatus.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+                    orderId = tx.receipt.events.NewOrder.returnValues.orderId;
+                    txInfo = <p>Order id: {orderId}</p>;
+                }
+
+                return (
+                    <MyListGroup.Row key={`txRowDiv-${hash}`}>
+                        {tx.event === "transactionHash" && (
+                            <LoadingPanel header={header} onDismiss={() => this.handleClose(tx.transactionHash)}>
+                                <p>
+                                    Transaction's sent to Ethereum network. Wait for confirmations. <br />
```
[the remark this answers is not in the document]
Comment (szerintedmi) [reply src/containers/app/EthereumTxStatus.js]: no need for escape character, I think it's an issue with github's jsx syntax highlighter

[the remark this answers is not in the document]
Comment (reyrodrigez) [reply src/containers/app/EthereumTxStatus.js]: nice! apparently if the file extension is `.jsx` the syntax highlighter gets it: https://github.com/github/linguist/issues/3044

**Pull request merged**